### PR TITLE
Ensure tab works on the flex grid control as a single control

### PR DESCRIPTION
**Pull request by kof**
Closes https://github.com/webstudio-is/webstudio-designer/issues/736

## Description

Before that you would have to tab through every knob from inside the flex grid control to get to the next element. For now we are making it just tabbable as a single thing. Later we can add arrow keys.

Also aligned colors with figma.

## Steps for reproduction

1. Make some element to flex container
2. Focus search on style panel
3. start tabbing, make sure you can tab through all controls in the flex UI, like you would expect

## Code Review

- [ ] hi @TrySound , I need you to do
  - conceptual review (architecture, feature-correctness)
  - detailed review (read every line)
  - test it on preview

## Before requesting a review

- [ ] made a self-review
- [ ] added inline comments where things may be not obvious (the "why", not "what")

## Before merging

- [ ] tested locally and on preview environment (preview dev login: 5de6)
- [ ] updated [test cases](https://github.com/webstudio-is/webstudio-designer/blob/main/apps/designer/docs/test-cases.md) document
- [ ] added tests
- [ ] if any new env variables are added, added them to `.env.example` and the `designer/env-check.js` if mandatory

### CODE REVIEW

File: apps/designer/app/designer/features/style-panel/sections/layout/shared/flex-grid.tsx [hunk 1]
```diff
@@ -44,18 +44,25 @@ export const FlexGrid = ({
...
-        border: "2px solid currentColor",
+        outlineStyle: "solid",
+        outlineWidth: styleSource === "default" ? 1 : 2,
+        outlineColor: color,
         alignItems: "center",
         gap: theme.spacing[1],
         gridTemplateColumns: "repeat(3, 1fr)",
         gridTemplateRows: "repeat(3, 1fr)",
         color,
+        "&:focus, &:focus-within": {
```
Comment (TrySound) [inline apps/designer/app/designer/features/style-panel/sections/layout/shared/flex-grid.tsx:62]: Just focus within should be enough, no?

File: apps/designer/app/designer/features/style-panel/sections/layout/shared/flex-grid.tsx [hunk 2]
```diff
@@ -44,18 +44,25 @@ export const FlexGrid = ({
...
+          outlineWidth: 2,
+          outlineColor: theme.colors.borderSetFlexUi,
```
Comment (TrySound) [inline apps/designer/app/designer/features/style-panel/sections/layout/shared/flex-grid.tsx:64]: focus and local colors are almost same. I didn't see the difference at first.

Comment (kof) [reply apps/designer/app/designer/features/style-panel/sections/layout/shared/flex-grid.tsx:64]: Yeah, will comment in figma, its a token

Comment (kof) [reply apps/designer/app/designer/features/style-panel/sections/layout/shared/flex-grid.tsx:64]: Actually it is all clear in figma, we just needed to upgrade colors to tokens

**Comment by kof** [comment]
Screenshot 2023-01-28 at 09 44 50

@taylornowotny whats the token that we used as a background in that middle knob on hover? seems I can't find it in figma, in code we use slate4

**Review by TrySound — APPROVED** [review]

**Comment by taylornowotny** [comment]
> Screenshot 2023-01-28 at 09 44 50
> 
> @taylornowotny whats the token that we used as a background in that middle knob on hover? seems I can't find it in figma, in code we use slate4

Screenshot 2023-01-28 at 9 37 34 AM

**Comment by taylornowotny** [comment]
> Screenshot 2023-01-28 at 09 44 50
> 
> @taylornowotny whats the token that we used as a background in that middle knob on hover? seems I can't find it in figma, in code we use slate4

That is not the hover effect that is in the design. The flex controls in general need a lot of work to match the design.

Screenshot 2023-01-28 at 9 39 04 AM

**Comment by kof** [comment]
I see, for the next iteration then

**Pull request merged**